nefertari_sqla/types.py

```diff
@@ -120,7 +120,10 @@ class Choice(types.TypeDecorator):
 
     def __init__(self, *args, **kwargs):
         self.choices = kwargs.pop('choices', ())
-        if not isinstance(self.choices, (list, tuple, set)):
+
+        if isinstance(self.choices, dict):
+            self.choices = list(self.choices.keys())
```

Comment:
Since `self.choices` is used only with the `in` operator, I think a `set` would be more fitting here.